src/ddscxx/include/org/eclipse/cyclonedds/pub/AnyDataWriterDelegate.hpp

```diff
@@ -105,29 +105,29 @@ class OMG_DDS_API AnyDataWriterDelegate : public org::eclipse::cyclonedds::core:
                           const dds::topic::TopicDescription& td);
 
 
-    inline void setCopyIn(org::eclipse::cyclonedds::topic::copyInFunction copyIn)
+    inline void setCopyIn(org::eclipse::cyclonedds::topic::copyInFunction _copyIn)
     {
-        this->copyIn = copyIn;
+        this->copyIn = _copyIn;
     }
 
     inline org::eclipse::cyclonedds::topic::copyInFunction getCopyIn()
     {
         return this->copyIn;
     }
 
-    inline void setSampleSize(size_t sampleSize)
+    inline void setSampleSize(size_t _sampleSize)
     {
-        this->sampleSize = sampleSize;
+        this->sampleSize = _sampleSize;
     }
 
     inline size_t getSampleSize()
     {
         return this->sampleSize;
     }
 
-    inline void setCopyOut(org::eclipse::cyclonedds::topic::copyOutFunction copyOut)
+    inline void setCopyOut(org::eclipse::cyclonedds::topic::copyOutFunction _copyOut)
     {
-        this->copyOut = copyOut;
+        this->copyOut = _copyOut;
     }
```

Comment:
Same.

Reply:
Removed this function, since it is not used anywhere.